manifests/virt-manifest.yaml.in

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: virt-manifest-service
+spec:
+  ports:
+    - port: 8186
+      targetPort: virt-manifest
+  externalIPs :
+    - "{{ master_ip }}"
+  selector:
+    app: virt-manifest
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: virt-manifest
+spec:
+  template:
+    metadata:
+      labels:
+        daemon: virt-manifest
+    spec:
+      hostNetwork: true
+      hostPID: true
+      hostIPC: true
+      securityContext:
+        runAsUser: 0
+      containers:
+      - name: virt-manifest
+        ports:
+          - containerPort: 8186
+            hostPort: 8186
+        image: {{ docker_prefix }}/virt-manifest:{{ docker_tag }}
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+          - name: libvirt-data
+            mountPath: /var/lib/libvirt
+          - name: libvirt-runtime
+            mountPath: /var/run/libvirt
+      - name: manifest-libvirtd
+        image: {{ docker_prefix }}/libvirt-kubevirt:{{ docker_tag }}
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          privileged: true
+          runAsUser: 0
+        env:
+        - name: LIBVIRTD_DISABLE_TCP
+          value: "true"
+        - name: LIBVIRTD_DEFAULT_NETWORK_DEVICE
+          value: eth1
+        volumeMounts:
```

Comment:
In order to slim this list down, we'd need to build a new docker container that doesn't assume those mount points are present (libvirtd.sh). I was trying to avoid doing that but it might be inevitable.